server/routes/induction/common/wantToAddQualificationsController.ts

```diff
@@ -40,6 +40,7 @@ export default abstract class WantToAddQualificationsController extends Inductio
       this.getBackLinkAriaText(req),
       wantToAddQualificationsForm,
       functionalSkills,
+      req.flash('errors'),
```

Comment:
For some reason errors were never added to the view model by the controller (though they were always in the view model class!)
(The cypress test that triggers validation highlighted this as a problem, as without this change it does not render the errors on screen)